skimage/data/__init__.py

```diff
@@ -12,13 +12,16 @@
 
 import numpy as _np
 
-from .. import data_dir
 from ..io import imread, use_plugin
 from .._shared._warnings import expected_warnings, warn
-from .. import img_as_bool
+from ..util.dtype import img_as_bool
 from ._binary_blobs import binary_blobs
 
-__all__ = ['load',
+import os.path as osp
+data_dir = osp.abspath(osp.dirname(__file__))
+
+__all__ = ['data_dir',
+           'load',
```

Comment:
Do we really need to expose `load` here?

Reply:
I don't know. It was there before.